cpp/dolfinx/io/VTKFile.cpp

```diff
@@ -520,11 +520,14 @@ void pvtu_write_function(std::size_t dim, std::size_t rank,
   data_array_node.append_attribute("NumberOfComponents")
       = (unsigned int)num_components;
 
+  std::string delimiter = "/";
+  std::string local_filename = filename.substr(filename.find(delimiter) + 1);
```

Comment:
This looks strange and platform-dependent. Any reason to not use `boost::filesystem`?